kuka_ros2_control_sim_support/launch/test_common_gazebo_classic_ros2_control.launch.py

```diff
@@ -0,0 +1,275 @@
+    robot_description_package = LaunchConfiguration("robot_description_package")
+    robot_description_macro_file = LaunchConfiguration("robot_description_macro_file")
+    robot_name = LaunchConfiguration("robot_name")
+    prefix = LaunchConfiguration("prefix")
+    robot_controller = LaunchConfiguration("robot_controller")
+    robot_controllers = PathJoinSubstitution(
+        [FindPackageShare(runtime_config_package), "config", controllers_file]
+    )
+    rviz_config_file = PathJoinSubstitution(
+        [FindPackageShare(runtime_config_package), "config", "view_robot.rviz"]
+    )
+
+    # Get URDF via xacro
+    robot_description_content = Command(
+        [
+            PathJoinSubstitution([FindExecutable(name="xacro")]),
+            " ",
+            PathJoinSubstitution(
+                [
+                    FindPackageShare("kuka_ros2_control_sim_support"),
+                    "urdf",
+                    "common_sim_kuka.xacro",
+                ]
+            ),
+            " ",
+            "prefix:=",
+            prefix,
+            " ",
+            "mock_sensor_commands:=false",
+            " ",
+            "sim_gazebo_classic:=true",
+            " ",
+            "sim_gazebo:=false",
+            " ",
+            "simulation_controllers:=",
+            robot_controllers,
+            " ",
+            "controllers_file:=",
+            controllers_file,
+            " ",
+            "robot_description_package:=",
+            robot_description_package,
+            " ",
+            "robot_description_macro_file:=",
+            robot_description_macro_file,
+            " ",
+            "robot_name:=",
+            robot_name,
+            " ",
+        ]
+    )
+    robot_description = {"robot_description": robot_description_content}
+
+    robot_state_pub_node = Node(
+        package="robot_state_publisher",
+        executable="robot_state_publisher",
+        output="both",
+        parameters=[robot_description],
+    )
+    rviz_node = Node(
+        package="rviz2",
+        executable="rviz2",
+        name="rviz2",
+        output="log",
+        arguments=["-d", rviz_config_file],
+    )
+
+    # # Gazebo node
+    gazebo = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [FindPackageShare("gazebo_ros"), "/launch", "/gazebo.launch.py"]
+        ),
+    )
+
+    # Spawn robot
+
+    gazebo_spawn_robot = Node(
+        package="gazebo_ros",
+        executable="spawn_entity.py",
+        arguments=["-topic", "robot_description", "-entity", "$ROBOT_NAME$"],
+        output="screen",
+    )
+
+    joint_state_broadcaster_spawner = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=["joint_state_broadcaster", "--controller-manager", "/controller_manager"],
+    )
+
+    robot_controllers = [robot_controller]
+    robot_controller_spawners = []
+    for controller in robot_controllers:
```

Comment:
robot_controllers variable already exists above, should this one be changed?
```suggestion
    robot_controllers_to_spawn = [robot_controller]
    robot_controller_spawners = []
    for controller in robot_controllers_to_spawn:
```